src/ucp/wireup/wireup_ep.h

```diff
@@ -35,6 +35,10 @@ struct ucp_wireup_ep {
     uct_ep_h                  aux_ep;        /**< Used to wireup the "real" endpoint */
     uct_ep_h                  sockaddr_ep;   /**< Used for client-server wireup */
     ucp_ep_h                  tmp_ep;        /**< Used by the client for local tls setup */
+    struct sockaddr_storage   cm_remote_sockaddr;  /**< sockaddr of the remote peer -
+                                                        used only on the client side
+                                                        in a client-server flow */
+    ucp_rsc_index_t           cm_idx;        /**< index of the used cm in the priority list */
```

Comment:
cm  index in the priority list which is **currently** in use for connection establishment on **client side**